File: Java Project  V1.7/backend-spring/src/main/java/com/emart/entity/Category.java
package com.emart.entity;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name="category_master")
@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
public class Category 
{
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer catmasterId;

	private String catId;

    private String subcatId;

    private String catName;

    private String catImagePath;

    private Boolean flag;
    public Integer getCatmasterId() 
    {
		return catmasterId;
	}
	public void setCatmasterId(Integer catmasterId) 
	{
		this.catmasterId = catmasterId;
	}
	public String getCatId() 
	{
		return catId;
	}
	public void setCatId(String catId) 
	{
		this.catId = catId;
	}
	public String getSubcatId() 
	{
		return subcatId;
	}
	public void setSubcatId(String subcatId) 
	{
		this.subcatId = subcatId;
	}
	public String getCatName() 
	{
		return catName;
	}
	public void setCatName(String catName) 
	{
		this.catName = catName;
	}
	public String getCatImagePath() 
	{
		return catImagePath;
	}
	public void setCatImagePath(String catImagePath) 
	{
		this.catImagePath = catImagePath;
	}
	public Boolean getFlag() 
	{
		return flag;
	}
	public void setFlag(Boolean flag) 
	{
		this.flag = flag;
	}

	
}

File: DotNet Project V2.0/backend-dotnet/src/Emart.Domain/Entities/Category.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Emart.Domain.Entities;

[Table("category_master")]
public class Category
{
    [Key]
    [Column("catmaster_id")]
    public int CatmasterId { get; set; }

    [Column("cat_id")]
    public string? CatId { get; set; }

    [Column("subcat_id")]
    public string? SubcatId { get; set; }

    [Column("cat_name")]
    public string? CatName { get; set; }

    [Column("cat_image_path")]
    public string? CatImagePath { get; set; }

    [Column("flag")]
    public bool? Flag { get; set; }
}
